src/index.html

```diff
@@ -3,11 +3,278 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Title</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>Museum</title>
+    <link rel="stylesheet" href="./styles/scss.scss">
   </head>
   <body>
-    <h1>Hello Mate Academy</h1>
+    <!DOCTYPE html>
+    <html lang="en">
+      <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Museum</title>
+        <link rel="stylesheet" href="./styles/main.scss">
+      </head>
+      <body>
+        <header class="header">
+          <section class="header__section section">
+            <div class="header__main">
+              <h1 class="header__title">ХУДОЖ<span class="header__title-span">НІЙ</span><br> МУЗЕЙ</h1>
```

Comment:
Перенось на нові рядки, так набагато простіше читати
```suggestion
              <h1 class="header__title">
                ХУДОЖ
                <span class="header__title-span">
                  НІЙ
                </span>
                <br>
                 МУЗЕЙ
               </h1>
```